http/src/client/interaction.rs

```diff
@@ -167,16 +167,13 @@ impl<'a> InteractionClient<'a> {
     /// the same guild will overwrite the old command. See [the discord docs]
     /// for more information.
```

Comment:
we should also add a note here about the new validation requirements, and in other places where applicable

Reply:
What kind of note would you like to add? Unless I forgot one, validation requirements are mentioned on methods each time it's applicable.

Reply:
after a second pass, it already looks fine. my bad